src/KeyValueStore.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018, 2019 Transposit Corporation. All Rights Reserved.
```

Comment:
only two years included if the file was created and then last updated in two different years. it's a best effort to keep this right.